modules/nf-core/mindagap/duplicatefinder/tests/main.nf.test

```diff
@@ -0,0 +1,35 @@
+nextflow_process {
+
+    name "Test Process MINDAGAP_DUPLICATEFINDER"
+    script "../main.nf"
+    config "./nextflow.config"
+    process "MINDAGAP_DUPLICATEFINDER"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mindagap"
+    tag "mindagap/duplicatefinder"
+
+    test("test_mindagap_duplicatefinder_spots") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test'], // meta map
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/molkart/test_data/input_data/spots.txt')
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.marked_dups_spots).match() },
+                { assert snapshot(process.out.versions).match("versions")}
```

Comment:
```suggestion
                { assert snapshot(process.out).match() }
```

Reply:
if all your out channels generate consistent md5...recommend capturing all of it in one go to keep code concise